charts/k8s-watcher/templates/configmap-metrics.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "k8s-watcher.name" . }}-metrics-config
+  {{- if hasKey .Values "namespace" }}
+  namespace: {{ .Values.namespace }}
+  {{- end }}
+data:
+  telegraf.conf: |
+    [global_tags]
+      AccountId = "${AccountId}"
+
+    [agent]
+      interval = "15s"
+      flush_interval = "1m"
+
+    [[outputs.timestream]]
+      region = "us-east-1"
+      access_key = "${TimestreamAccessKey}"
+      secret_key = "${TimestreamSecretKey}"
+
+      database_name = "prod-metrics-db"
```

Comment:
there are many configuration details hardcoded here which might lock us in the future if we'd like to change them
I'm referring to database name, region, retention periods, scrape interval and flush interval etc

IMO we should be receiving these from the collector endpoint for metrics configuration the same way we do with the api keys

not a lot of work but will allow us to be more agile if we need to

Reply:
will do

Reply:
done